server/events/event_parser.go

```diff
@@ -610,6 +625,13 @@ func (e *EventParser) ParseGithubRepo(ghRepo *github.Repository) (models.Repo, e
 	return models.NewRepo(models.Github, ghRepo.GetFullName(), ghRepo.GetCloneURL(), e.GithubUser, e.GithubToken)
 }
 
+// ParseGithubRepo parses the response from the GitHub API endpoint that
+// returns a repo into the Atlantis model.
+// See EventParsing for return value docs.
+func (e *EventParser) ParseGiteaRepo(repo giteasdk.Repository) (models.Repo, error) {
+	return models.NewRepo(models.Gitea, repo.FullName, repo.CloneURL, e.GithubUser, e.GithubToken)
```

Comment:
Passed incorrect arguments: 

`GithubUser` -> `GiteaUser`
`GithubToken` -> `GiteaToken`

Reply:
And variable eventParser in `server/server.go#546` file missing those value above.

Reply:
Corrected. I'll try to get screenshots of a private repo later tonight.